### DISCUSSION TITLE
How does podsync download files on a slow computer?

### DISCUSSION THREAD

**Discussion by ivanbili** [mxpv/podsync#241] [Q&A]
I am running podsync on a very slow computer (Raspberry Pi Zero), and my downloads don't seem to be finishing. I am wondering, how does podsync schedule downloads? It uses youtube-dl, but what if it takes a long time to download and convert? Will it wait for it to finish before downloading the next item or will it cancel the download every time it should run?

I am using a cron schedule like this:

cron_schedule = "0 2 * * *"

so the downloads should all start at 2am. I think.

**Comment by wanghaisheng** [comment]
run a simple youtube-dl container image first to check whether there is download problem actually.
then came to convert process
